### ISSUE TITLE
Browser Sidebar not Styled on Mac

### ISSUE THREAD

**Issue by Kosezu** [krassowski/Anki-Night-Mode#95]
#### Problem description

The sidebar of the browser is no longer styled in 2.1.17 on a MacOS.

The rest of the browser is styled correctly.

#### Checklist

*Please put x inside brackets if following sentences apply to you:*

 - [X] I verified that I use the lastest add-on version,
 - [X] I verified that I use the lastest Anki version,
 - [X] I tried to disable other add-ons to find the culprit,
 - [X] I checked if anyone reported this problem before, as well as Known issues section on add-ons page,
 - [X] My issue is solved when holding shift while starting Anki.
 
#### Information about your Anki Setup

Latest Anki and latest Night Mode add-on.

*Please open Anki, go to Help → About, look-up two lines, starting with `Version:` and `Qt` and copy-paste them below:*

```
Version: 2.1.17 (c69ccb50)
Qt: 5.14.0 PyQt 5.13.2
```

*Please got to Night Mode → About now and copy-paste the first line from the pop-up window below:*

```
Night Mode 2.3.0
```

*Please fill in your OS details (Linux/Mac/Windows):*

```
OS: macOS Catalina 10.15 
```
 
 
*If you have a compatibility problem with a specific add-on or add-ons, please paste the add-on page links below:*
 - [Other add-on name](https://)
</details>

#### Backtrace (if any)

<details>

*If you had an error which resulted in producing exception with a backtrace, please paste the backtrace below:*

```python"
[paste your backtrace in here if you got any; otherwise feel free to remove this part]
```

</details>
<img width="1453" alt="Screen Shot 2020-01-14 at 17 07 56" src="https://user-images.githubusercontent.com/35477971/72325480-88006680-36f0-11ea-982c-ddea16417842.png">

**Comment by blaxwhix** [comment]
This issue is also present in Linux as well.

**Comment by ran-dall** [comment]
This is happening to me too. I've had to disable the styling of the browser altogether.

**Comment by krassowski** [comment]
Closing as duplicate of #90 and fixed with #94 in 2.3.1. Please update, test and reopen if needed. Have a nice weekend with Anki!
